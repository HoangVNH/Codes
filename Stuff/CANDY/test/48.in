10
79
VXVXVVVXVVDDXVVXXVXXXVVDVXVDDVDVDDDDDXXXDDDDXDVVVXDVDDVDVVXXVVXVVVVDVXVXXXDDXXX
97
DVDXDXVXVXDDDXXXXXXVVVXXVXVDXVVXDDDDXDDVXDXXXXXVVDDDXXDDVXDXVDXDDDDXXDVXXDVXDXVVXXVXXDVDDVVXDDXXX
70
DDDVVXVVVDDDDDDVDXVDDXDXVVXDDXXVXDDDXDDDVVXVDVVXVXXXDXVXDXXXXVXDXXDVVV
59
VXVDVVVVVVVXVVDDVVXDVXVVVDXXXVXVDVDDDDXXDVDXXDDVDXVDDDVVDVD
56
VXXXDXXDVXDXDVVXXXDVDXDVDVVVVDXXDVDDVXXXDDDVXXVDVXDVXDVD
85
DDXVVDXDVVXDXXVXXXVDDXXVXDVVVXXDVXXXXDVVDXVDVDXDXVVVVVVXVXVXXXXDXVXVXDXDXVDXVVDVDXXXV
53
VXVDVDVDVXDDDDDXDXXDXXXDXVXVDXDVXXVVXVVDVXXXVDXXDXVDV
69
VDDDXXVDDXVDVDDDDXXVVXVDVVVDVVXVVVVXVVVXVXXXXXXXVXXVXXDDVVXXXDVDXDDVD
87
VVXVXXXVVDXXVXXDVDXXDDDXXDXDXDDVXDDDDVDVDXDVXXVDVDXXXDVDDDXVVDDDVDXVXXVVXVVXDXVVVXVDXVX
60
XVDXDDDDDDDVDDDXVVDDDXXVVVXVDVDDXDXDXVVVDVDXXDDDDXXDVDVXVVXX
