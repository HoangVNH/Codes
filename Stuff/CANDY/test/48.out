77
95
69
58
55
84
52
68
85
59
